doc/api/process.md

```diff
@@ -2305,6 +2305,222 @@ import { loadEnvFile } from 'node:process';
+   assert.strictEqual(process.env.COMMENTS, undefined);
+   assert.strictEqual(process.env.INLINE_COMMENTS, 'inline comments');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.COMMENTS, undefined);
+   assert.strictEqual(process.env.INLINE_COMMENTS, 'inline comments');
+   ```
+
+3. **Whitespace handling**:
+
+   * Leading and trailing whitespaces around keys and values are ignored unless
+     they are enclosed within quotes.
+
+   ```bash
+         SPACED_KEY = parsed
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.SPACED_KEY, 'parsed');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.SPACED_KEY, 'parsed');
+   ```
+
+4. **Empty values**:
+
+   * Variables with no value assigned (just a key followed by `=`) are set to
+     an empty string.
+
+   ```bash
+     EMPTY=
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.EMPTY, '');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.EMPTY, '');
+   ```
+
+5. **Quoted values**:
+
+   * Single (`'`), double (`"`), and backtick (`` ` ``) quotes are recognized.
+     The content within the quotes is taken as is, including leading and
+     trailing spaces.
+   * Quotes within a different quote type are preserved.
+
+   ```bash
+     FIRST_NAME=' John '
+     MIXED_QUOTES="Say 'Hello!'"
+     BACKTICK_IN_QUOTES="Using `backticks` in double quotes"
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.FIRST_NAME, ' John ');
+   assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+   assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                      'Using `backticks` in double quotes');
+   ```
+
+   ```mjs
+   import assert from 'node:assert';
+   import process from 'node:process';
+   assert.strictEqual(process.env.FIRST_NAME, ' John ');
+   assert.strictEqual(process.env.MIXED_QUOTES, "Say 'Hello!'");
+   assert.strictEqual(process.env.BACKTICK_IN_QUOTES,
+                      'Using `backticks` in double quotes');
+   ```
+
+6. **Multiline values**:
+
+   * Values enclosed in double, single, or backtick quotes that span multiple
+     lines are accepted and stored with newline characters.
+
+   ```bash
+    MULTI_DOUBLE_QUOTED="double
+    quoted"
+
+    MULTI_SINGLE_QUOTED='single
+    quoted'
+
+    MULTI_BACKTICKED=`this
+    "multiline"
+    value`
+   ```
+
+   ```cjs
+   const assert = require('node:assert');
+   const process = require('node:process');
+   assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'double\nquoted');
```

Comment:
You need to escape all \ characters

```suggestion
   assert.strictEqual(process.env.MULTI_DOUBLE_QUOTED, 'double\\nquoted');
```

Reply:
In fact, I got the values from test files https://github.com/nodejs/node/blob/main/test/parallel/test-dotenv.js#L72